src/journal/JournalRecorder.h

```diff
@@ -38,6 +38,7 @@ class JournalRecorder {
 
 private:
   typedef std::map<uint8_t, ObjectRecorderPtr> ObjectRecorderPtrs;
+  typedef std::vector<std::unique_lock<ceph::mutex>> Lockers;
```

Comment:
compiler error would appear to be emanating from this line?

Reply:
@smithfarm in nautilus we still use `Mutex` instead of `ceph::mutex`. So it should be adjusted accordingly

Reply:
@trociny Like this?

```
-  typedef std::vector<std::unique_lock<ceph::mutex>> Lockers;
+  typedef std::vector<std::unique_lock<Mutex>> Lockers;
```

Reply:
I think it should be something like this:
```
typedef std::vector<Mutex::Locker> Lockers;
```

Reply:
@trociny I changed it to

```
typedef std::vector<Mutex::Locker> Lockers;
```

but still no joy.